src/main/java/uk/gov/hmcts/reform/ccd/documentam/controller/advice/CaseDocumentControllerAdvice.java

```diff
@@ -57,31 +59,52 @@
     @ExceptionHandler(RequiredFieldMissingException.class)
     protected ResponseEntity<Object> handleRequiredFieldMissingException(RequiredFieldMissingException exception) {
         return errorDetailsResponseEntity(exception, BAD_REQUEST,
-            ErrorConstants.INVALID_REQUEST.getErrorCode(), ErrorConstants.INVALID_REQUEST.getErrorMessage());
+            ErrorConstants.BAD_REQUEST.getErrorCode(), ErrorConstants.BAD_REQUEST.getErrorMessage());
```

Comment:
i don't see a point of these custom errors like `INVALID_REQUEST` `RESOURCE_NOT_FOUND`.  It has already standard ones "BAD_REQUEST" and "NOT_FOUND"